### PR TITLE
Increase robustness of performance tests.

### PR DESCRIPTION
**Pull request by seanparsons**
**Problem:**
Recent changes appear to have made the performance tests flakier to run.

**Fix:**
Individual tests now load into their own unique project ID, to try and avoid any issues with carrying over anything between tests. Some extra leeway has been added to the flow of loading a project as well.

**Commit Details:**
- Loading a project now increments the project ID.
- An extra `await wait(...)` was added at the end of `loadProject` to give
  the editor an extra chance to finish up any loading.
- `loadProject` now attempts 3 times to load the given project.
- Added a missing log line for `calculatePi`.

### CODE REVIEW

File: puppeteer-tests/src/performance-test.ts [hunk 1]
```diff
@@ -180,7 +181,7 @@ export const testPerformanceInner = async function (url: string): Promise<Perfor
     selectionResult = await testSelectionPerformance(page)
     resizeResult = await testResizePerformance(page)
     scrollResult = await testScrollingPerformance(page)
-    absoluteMoveResult = await testAbsoluteMovePerformance(page)
+    //absoluteMoveResult = await testAbsoluteMovePerformance(page)
```
[the remark this answers is not in the document]
Comment (seanparsons) [reply puppeteer-tests/src/performance-test.ts:184]: Currently yes, to test how this runs on the CI environment.

File: puppeteer-tests/src/performance-test.ts [hunk 2]
```diff
@@ -98,6 +98,7 @@ function calculatePi(accuracy: number): number {
 }
 
 function timeBasicCalc(): FrameResult {
+  console.log('Test Basic Calc')
```
[the remark this answers is not in the document]
Comment (Rheeseyb) [reply puppeteer-tests/src/performance-test.ts:101]: Ignore that

**Review by Rheeseyb — APPROVED** [review]

File: puppeteer-tests/src/performance-test.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (seanparsons) [reply puppeteer-tests/src/performance-test.ts:101]: Ignored!

[the remark this answers is not in the document]
Comment (Rheeseyb) [reply puppeteer-tests/src/performance-test.ts:101]: Thank you for your cooperation

**Review by balazsbajorics — APPROVED** [review]

**Pull request merged**